src/tribler-core/tribler_core/modules/tunnel/community/dispatcher.py

```diff
@@ -87,7 +87,7 @@ def on_socks5_udp_data(self, udp_connection, request):
 
         self._logger.debug("Sending data over circuit destined for %r:%r", *request.destination)
         self.circuit_id_to_connection[circuit.circuit_id] = udp_connection.socksconnection
-        self.tunnel_community.send_data([circuit.peer.address], circuit.circuit_id, request.destination,
+        self.tunnel_community.send_data(circuit.peer, circuit.circuit_id, request.destination,
```

Comment:
Are you sure that this working correctly, given that `send_packet` in `TunnelCommunity` runs a for-loop? It doesn't get a list anymore now.

Reply:
Yesterday I changed all the send functions in the `TunnelCommunity` such that they don't take a list anymore (see https://github.com/Tribler/py-ipv8/commit/8c6f4a8bf45d490748859139c1c82b5c3a7f15ae).